nengo/tests/test_learning_rules.py

```diff
@@ -33,7 +33,9 @@ def _test_pes(Simulator, nl, plt, seed,
 
         conn = nengo.Connection(pre, post,
                                 function=function, transform=transform,
-                                learning_rule_type=PES(rate))
+                                learning_rule_type=PES(
```

Comment:
Maybe define the rule outside of the `nengo.Connection` for the sake of readability?